footer.php

```diff
@@ -2,20 +2,18 @@
 		</div><!-- container footer-widgets -->
 </div><!-- wrapper container -->
 
-		<div class="container" id="foot">
-			<div class="row">
+		<footer class="container" id="foot">
+			<nav aria-labelledby="Connect with KBCS" class="row">
```

Comment:
Close but not quite there! aria-labelledby should include the ID of an element that labels the `<nav>`. If you would like to provide a label directly, using `aria-label` is the way to go instead.

## Example of aria-labelledby:
```html
<nav aria-labelledby="nav-heading">
    <h2 id="nav-heading">Related Programs</h2>
    ...
</nav>
```

## Example of aria-label
```html
<nav aria-label="Related Programs">
    ...
</nav>
```

Reply:
Thanks for the detailed explanation, I have resolved this issue using aria-label instead.